integration_tests/pages/homepage.js

```diff
@@ -2,16 +2,16 @@ const page = require('./page')
 
 const homePage = () =>
   page('Manage restricted patients', {
-    loggedInName: () => cy.get('[data-test="logged-in-name"]'),
```

Comment:
not sure much of this needs testing anymore since it is served up as static content by wiremock